qreu/sendcontext.py

```diff
@@ -92,5 +92,7 @@ def sendmail(self, mail):
         from_mail = mail.from_
         if isinstance(mail.from_, Address):
             from_mail = from_mail.address
-        self._connection.sendmail(from_mail, mail.recipients, mail.mime_string)
+        self._connection.sendmail(
+            from_mail, mail.recipients + [','.join(mail.bccs)],
```

Comment:
It (`mail.recipients + [','.join(mail.bccs)]`) should be added on the recipients properties